Pre-generate the JavaScript messages during installation by postinstall

minggangw opened the pull request:
To accelerate the speed of first-time start up, we better pre-generate
the messages during installation. Thus we do not have to wait the work
to be done when the module is initialized.

Fix #356

qiuzhong commented:
Tested this patch:
* First time `npm install`, `generated` directory and message files are generated.
* Remove `generated` and run `npm install`, message files are generated again.
* Remove `generated` and calling `rclnodejs.init()`, message files are generated as well.

So I think this patch works.

qiuzhong commented:
@minggangw , I have another concern: if this patch is merged with `actionlib` feature, will it still work?

minggangw commented:
This patch will be merged into `develop` branch. When the actionlib feature gets finished, the `actionlib` branch will merged with `develop`. This function will not be affected, which means all .msg/.srv/.action will be processed during installation.

minggangw commented:
We have to wait the latest package for xenial to fix the error on ci https://github.com/ros2/ci/issues/187

Commit:
Pre-generate the JavaScript messages during installation by postinstall

To accelerate the speed of first-time start up, we better pre-generate
the messages during installation. Thus we do not have to wait the work
to be done when the module is initialized.

Fix #356
